secrets.txt

```diff
@@ -0,0 +1,2 @@
+MAIL_USERNAME=sowecthal@gmail.com
```

Comment:
Why secrets are not encrypted? Now anybody would be able to use your password. Pls reorganize your git histroy to not have such unencrypted secrets anywhere in git commits

Reply:
Fixed